lib/active_fedora/schema_indexing_strategy.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveFedora
+  ##
+  # An application strategy to also apply solr indexes for each property.
+  # @note If how a field is indexed changes based on property, this would be a
+  #   good place to define that.
+  class SchemaIndexingStrategy
```

Comment:
Should this extend AT::ExtensionStrategy? I know you are overriding everything, but it seems useful at least as documentation.